apps/backend/core/configuration/validation/validators/platform_validators.py

```
@@ -0,0 +1,233 @@
+        if not self.DSN_PATTERN.match(dsn):
+            return self.create_result(
+                status=ValidationStatus.FAIL,
+                message="Invalid DSN format",
+                error="DSN must match pattern: https://[key]@[host]/[project_id]",
+                latency_ms=(time.time() - start_time) * 1000,
+            )
+
+        # Extract project ID from DSN
+        try:
+            project_id = dsn.split("/")[-1]
+        except Exception:
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)